test/common/test_common.cpp

```diff
@@ -531,6 +531,55 @@ TEST (PCL, HasField)
   EXPECT_FALSE ((pcl::traits::has_label<pcl::Normal>::value));
 }
 
+TEST (PCL, GetMinMax3D)
+{
+  pcl::PointCloud<pcl::PointXYZ> cloud;
+  cloud.emplace_back ( 0.0f,      0.0f,  0.0f);
+  cloud.emplace_back (10.0f, -10000.0f,  1.0f);
+  cloud.emplace_back ( 5.0f,      5.0f,  0.0f);
+  cloud.emplace_back (-5.0f,      0.0f, -0.5f);
+
+  pcl::PointIndices pindices;
```

Comment:
Shall we move the indices variables closer to the tests?

Reply:
My idea was to first create all the needed objects, and then do the tests. But I can move it around if you like that better.

Reply:
Please do. Since not all the tests use the same variables, it'll be easier to read

Reply:
Done